packages/flutter_tools/lib/src/base/analyze_size.dart

```diff
@@ -17,12 +18,15 @@ class SizeAnalyzer {
   SizeAnalyzer({
     @required this.fileSystem,
     @required this.logger,
+    // TODO(jonahwilliams): migrate to required once this has rolled into google3.
+    Usage flutterUsage,
     this.appFilenamePattern = 'libapp.so',
-  });
+  }) : _flutterUsage = flutterUsage;
 
   final FileSystem fileSystem;
```

Comment:
ultra nit: since we're doing construction injection now and this file is new, should all these be private?